docs/concepts/service-catalog/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Service Catalog
+approvers:
+- chenopis
+---
+
+{% capture overview %}
+{% glossary_definition term_id="service-catalog" length="all" %}
+
+{% endcapture %}
+
+
+{% capture body %}
+## Example use case
+
+An {% glossary_tooltip text="Application Developer" term_id="application-developer" %} wants to use message queuing as part of their application running in a Kubernetes cluster.
+However, they do not want to deal with the overhead of setting such a service up and administrating it themselves.
+Fortunately, there is a cloud provider that offers message queuing as a *Managed Service* through their *Service Broker*.
+
+A *Service Broker*, as defined by the [Open Service Broker API spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md), is an endpoint for a set of Managed Services offered and maintained by a third-party, which could be a cloud provider such as AWS, GCP, or Azure.
```

Comment:
This information is a little too important to be contained within "Example Use Case"--it's core to understanding the feature.

Reply:
Ok, I'm moving this up into the Overview section.